examples/ex1.cpp

```diff
@@ -143,7 +143,7 @@ int main(int argc, char *argv[])
    bool delete_fec;
    if (order > 0)
    {
-      fec = new H1_FECollection(order, dim);
+      fec = FECollection::NewH1(order, dim, mesh.NURBSext);
```

Comment:
This is fine, but we might consider a member function in `Mesh`:
`bool isNURBS() const { return NURBSext; }`

Reply:
Added this function to the mesh object.